### DISCUSSION TITLE
SSH to github connection times out for no reason.

### DISCUSSION THREAD

**Discussion by carlesfito** [community/community#71736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

#What I'm trying to do
I've been accessing github from my pc with ssh keys for a long time. 
I'm currently trying to run some of my repositories on a **fresh** install of Ubuntu 22.04 under the same network. 

#The problem I have
The issue I have is that when I try to connect to github from the Ubuntu server I get the following:

`ssh -vT git@github.com
OpenSSH_8.9p1 Ubuntu-3ubuntu0.4, OpenSSL 3.0.2 15 Mar 2022
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: include /etc/ssh/ssh_config.d/*.conf matched no files
debug1: /etc/ssh/ssh_config line 21: Applying options for *
debug1: Connecting to github.com [140.82.121.4] port 22.
debug1: connect to address 140.82.121.4 port 22: Connection timed out
ssh: connect to host github.com port 22: Connection timed out
`

# Things that I tried so far
When I ping github.com It works. I'm using the same DNS as the computer that works.

I added the keys to github and what works for me on my Windows Desktop doesn't seem to work for the Ubuntu server, I even tried to use the same key that is being used for the former.
I tried to have no key and wrong key in the PC that works and it doesn't timeout either, it just says that my key is not valid.

My firewall (ufw) is disabled.

I tried connecting over the 443 port aswell as some other discussions sugested, no luck.

**Comment by krabelize** [comment]
Do you have a VPN tunnel configured? This seems like a network stability issue.
